packages/app/src/components/App.js

```diff
@@ -114,6 +114,9 @@ export class App extends Component {
             fromActions.fromSettings.tAddSettings(userSettings)
         )
         store.dispatch(fromActions.fromUser.acReceivedUser(d2.currentUser))
+        store.dispatch(
+            fromActions.fromUser.tLoadUserAuthority('F_VIEW_UNAPPROVED_DATA')
```

Comment:
`const` rather than string?

Reply:
Defined the const in the option component and imported in App.js.